Cargo.toml

```diff
@@ -13,20 +13,24 @@ members = ["examples"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["async-tungstenite", "cynic"]
+default = ["async-tungstenite", "ws_stream_wasm", "cynic"]
 
 [dependencies]
-async_executors = { git = "https://github.com/obmarg/async_executors.git", branch = "tokio-context" }
-futures = { version = "0.3"}
+async_executors = { version = "0.4.2" }
+futures = { version = "0.3" }
 pin-project = "1"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 thiserror = "1.0"
 uuid = { version = "0.8", features = ["v4"] }
+pin-project-lite = "0.2.7"
+pharos = { version = "0.5.2" }
```

Comment:
Might be good to stick these behind the `ws_stream_wasm` feature flag.

Reply:
Hmm that's not as easy. Then we need custom feature names. Which is okay, but then we should think about changing the ones for async-std/tungstenite as well :)